### PR TITLE
EVO-1074 render message names as links to user profiles

### PR DESCRIPTION
**Pull request by Sofacosta**

### CODE REVIEW

File: src/routes/Messages/Header/Header.js [hunk 1]
```diff
@@ -54,9 +53,10 @@ export default class Header extends React.Component {
   render () {
     const { showAll } = this.state
     const { pending } = this.props
-    const otherParticipants = this.getOthers(this.props)
+    const participants = get('participants', this.props.messageThread) || []
+    const otherParticipants = this.getOthers(this.props.currentUser, participants)
```
Comment (tibetsprague) [inline src/routes/Messages/Header/Header.js:57]: we generally like to pull out all the variables from this.props above, so add currentUser to const { pending } = this.props

Comment (tibetsprague) [reply src/routes/Messages/Header/Header.js:57]: and messageThread too

**Review by tibetsprague — CHANGES_REQUESTED** [review]
This looks great! Only thing is there are some compile/lint issues, make sure to check the running server console to see them before you push, or run yarn lint.

**Review by lorenjohnson — APPROVED** [review]
This looks great! Good work @Sofacosta 👍🏼

**Comment by Sofacosta** [comment]
It looks like the linter issues have been taken care of @tibetsprague

**Comment by tibetsprague** [comment]
yup looks good!

**Review by tibetsprague — APPROVED** [review]

**Pull request merged**